### PR TITLE
Make audio and video content visible in dialog

### PR DESCRIPTION
**Pull request by henriettemoe**
Makes audio and video visible in dialog. Audio can be played in dialog, video still needs some adjustments.

### CODE REVIEW

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 1]
```diff
@@ -104,27 +103,27 @@ const tabItems = (item: CommonItemType): JSX.Element[] => {
...
+              path: dialog.video[0].path,
+              mime: dialog.video[0].mime,
+              copyright: dialog.video[0].copyright,
             }}
           />
         </Content>,
       )
     : null;
-  dialog?.audio && dialog.audio.file && dialog.audio.file.path
+  dialog?.audio && dialog.audio.audioFile && dialog.audio.audioFile[0].path
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:119]:
```suggestion
  dialog?.audio?.audioFile?.[0]?.path
```

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 2]
```diff
@@ -64,14 +63,14 @@ const tabLabelItems = (
...
-  dialog?.video && dialog.video.path
+  dialog?.video && dialog.video[0].path
     ? items.push(
         <Trigger key="video" className={styles.trigger} value="Video">
           {translation.video}
         </Trigger>,
       )
     : null;
-  dialog?.audio && dialog.audio.file && dialog.audio.file.path
+  dialog?.audio && dialog.audio.audioFile && dialog.audio.audioFile[0].path
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:73]:
```suggestion
  dialog?.audio?.audioFile?.[0].path
```

Comment (boyum) [reply src/components/Dialog-Window/Tabs/DialogTabs.tsx:73]: Perhaps we should have `hasVideo` and `hasAudio` variables.

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 3]
```diff
@@ -104,27 +103,27 @@ const tabItems = (item: CommonItemType): JSX.Element[] => {
...
         <Content key="audio" value="Audio">
           <DialogAudio
             audioTrack={{
-              path: dialog.audio.file.path,
-              mime: dialog.audio.file.mime,
-              copyright: dialog.audio.file.copyright,
+              path: dialog.audio.audioFile[0].path,
+              mime: dialog.audio.audioFile[0].mime,
+              copyright: dialog.audio.audioFile[0].copyright,
             }}
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:127]: Change this for `audioTrack={dialog.audio.audioFile[0]}`

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 4]
```diff
@@ -104,27 +103,27 @@ const tabItems = (item: CommonItemType): JSX.Element[] => {
...
         <Content key="video" value="Video">
           <DialogVideo
             video={{
-              path: dialog.video.path,
-              mime: dialog.video.mime,
-              copyright: dialog.video.copyright,
+              path: dialog.video[0].path,
+              mime: dialog.video[0].mime,
+              copyright: dialog.video[0].copyright,
             }}
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:114]: Change this for audioTrack={dialog.video.videoFile[0]}

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 5]
```diff
@@ -104,27 +103,27 @@ const tabItems = (item: CommonItemType): JSX.Element[] => {
         </Content>,
       )
     : null;
-  dialog?.video && dialog.video.path
+  dialog?.video && dialog.video[0].path
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:106]:
```suggestion
  dialog?.video?.[0].path
```

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx [hunk 6]
```diff
@@ -64,14 +63,14 @@ const tabLabelItems = (
         </Trigger>,
       )
     : null;
-  dialog?.video && dialog.video.path
+  dialog?.video && dialog.video[0].path
```
Comment (boyum) [inline src/components/Dialog-Window/Tabs/DialogTabs.tsx:66]:
```suggestion
  dialog?.video?.[0].path
```

**Review by boyum — APPROVED** [review]
Nice!

File: src/components/Dialog-Window/Tabs/DialogTabs.tsx (hunk 2 above)
Comment (henriettemoe) [reply src/components/Dialog-Window/Tabs/DialogTabs.tsx]: If so, we will need to add the variables in two places anyway

**Review by boyum — APPROVED** [review]

**Pull request merged**